dataplane/ebpf/src/main.rs

```diff
@@ -45,28 +45,26 @@ unsafe fn ptr_at<T>(ctx: &TcContext, offset: usize) -> Result<*mut T, i64> {
     Ok((start + offset) as *mut T)
 }
 
-#[map(name = "BACKENDS")] 
+#[map(name = "BACKENDS")]
```

Comment:
Eventually we'll probably want this pinned with 

`    HashMap::<BackendKey, Backend>::pinned(128, 0);`